### ISSUE TITLE
Page specific options v3.6.8

### ISSUE THREAD

**Issue by micmath** [putyourlightson/craft-blitz#459]
I have a twig template that currently just shows JSON content returned from a module. The page specific options that I'm using seem to have no effect: the output comments are being added, the page does not seem to expire ever. I'm using version 3.6.8.

I can't see what I'm doing wrong, even after studying the documentation. Am I misunderstanding something?

```
{% do craft.blitz.options({
    cachingEnabled: true,
    cacheDuration: 'PT2M',
    outputComments: false,
    tags: ['api', 'event-manager']
}) %}{{ craft.mymodule.getData(id) | raw }}
```

**Comment by bencroker** [comment]
Which output comment is appearing exactly? And does refreshing the cache have any effect on this?

**Comment by micmath** [comment]
Here is an example of what I see in my web browser when I first hit the uncached endpoint handled by the twig template in my original post:

```
{"_date":"2023-01-13T09:45:53+00:00"}<!-- Served by Blitz on 2023-01-13T09:45:53+00:00 -->
```

If I reload that same page the comment goes away:

```
{"_date":"2023-01-13T09:45:53+00:00"}
```
Notice that the second (and subsequent) response is indeed cached, you can see that the `_date` value is the same.

This is only test code for debugging purposes of course. But if I follow the documentation, as shown in my original post, `outputComments: false`, I would expect that there should _never_ be a comment in the output for that template.

---

Regarding your second question, if I refresh the cache, like so:

```
$ ./craft blitz/cache/refresh-tagged api           
  Tagged cache successfully refreshed.
```

Then the situation is reset, and I again get the comment on first load:

```
{"_date":"2023-01-13T09:49:11+00:00"}<!-- Served by Blitz on 2023-01-13T09:49:11+00:00 -->
```

and it again goes away on subsequent loads:

```
{"_date":"2023-01-13T09:49:11+00:00"}
```

**Comment by bencroker** [comment]
Thanks for the details. Fixed in https://github.com/putyourlightson/craft-blitz/commit/e6efa943a2ff87b3d1f9e2af41a9a1f3b55f81e1 and released in version 3.12.8. It will also be released in the upcoming 4.3.0.

**Comment by bencroker** [comment]
Just released in version 4.3.0.
